src/features/user/DonationReceipt/microComponents/DonorContactForm.tsx

```diff
@@ -79,7 +77,7 @@ const DonorContactForm = ({
   const tAddressManagement = useTranslations('EditProfile.addressManagement');
   const t = useTranslations('DonationReceipt');
   const { user, contextLoaded, token, setUser, logoutUser } = useUserProps();
-  if (!user || !donationReceiptData) return null;
+  if (!user) return null;
```

Comment:
Does this make it possible to access this page without donationReceiptData? Does #2411 take care of this?